package.json

```diff
@@ -373,5 +373,8 @@
             "@hapi/address": "Version 2.1.2+ requires a polyfill for TextEncoder. Pinning to version 2.1.1 prevents IE11 from breaking.",
             "react-tether": "Version 2.x has too many breaking changes and requires forwardRef on all components"
         }
+    },
+    "resolutions": {
```

Comment:
There is a bug at 18.0.0 that requires this resolution otherwise storybook breaks